Bio/motifs/minimal.py

```diff
@@ -0,0 +1,162 @@
+# By Ariel Aptekmann 2017
+# Adapted from Bio.motifs 2008 by Bartek Wilczynski
+# Adapted from  Bio.MEME.Parser by Jason A. Hackney.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+from __future__ import print_function
+
+from Bio.Alphabet import IUPAC
+from Bio import Seq
+from Bio import motifs
+import math
+
+def read(handle):
+    """Parses the text output of the MEME program into a meme.Record object.
+    Example:
+    >>> from Bio.motifs import meme
+    >>> with open("meme.output.txt") as f:
+    ...     record = meme.read(f)
+    >>> for motif in record:
+    ...     for instance in motif.instances:
+    ...         print(instance.motif_name, instance.sequence_name, instance.strand, instance.pvalue)
+    """
+    motif_number = 0	
+    record = Record()
+    __read_version(record, handle)
+    __read_alphabet(record, handle)
+    __read_background(record, handle)
+
+    while True:
+    	for line in handle:
+		if line.startswith('MOTIF'): 
+	    		break
+	else :
+		return record
+	name  = line.split()[1]
+	motif_number +=1
+        length, num_occurrences, evalue  = __read_motif_statistics(line,handle)
+        counts = __read_lpm(line,handle)
+	#{'A': 0.25, 'C': 0.25, 'T': 0.25, 'G': 0.25}
+        motif = motifs.Motif(alphabet=record.alphabet,counts=counts)
+	motif.background = record.background
+        motif.length = length
+        motif.num_occurrences = num_occurrences
+        motif.evalue = evalue
+        motif.name = name
+        record.append(motif)
+        assert len(record) == motif_number
+	
+
+    return record
+
+
+class Record(list):
+    """A class for holding the results of a minimal MEME run.  """
```

Comment:
Can you remove the trailing whitespace after the full stop please? This will be failing one of the flake8 docstring checks we run on TravisCI as part of our style checks.

Reply:
Ok